k8s/cassandra/apptest/deployer/manifest/tester.yaml.template

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "${APP_INSTANCE_NAME}-tester"
+  labels:
+    app.kubernetes.io/name: "${APP_INSTANCE_NAME}"
+  annotations:
+    marketplace.cloud.google.com/verification: test
+spec:
+  serviceAccountName: ${APP_INSTANCE_NAME}-deployer-sa
```

Comment:
This is fragile as we rely on an implementation detail, which is the name of the deployer service account as well as its permissions.

Ideally the test `schema.yaml` should declare a `SERVICE_ACCOUNT` property, similar to the main `schema.yaml`, and use it here.

We however currently do not support this type of property in test `schema.yaml`. I filed https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/issues/275. This is OK for now, but please also file an issue and add a TODO here.

Reply:
Added TODO. It seems that you have already filled and issue for that.